scripts/ci/run-integration-tests.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/bash -ue
+
+cd /tmp/rchain/integration-testing
+python3 -m pip install pipenv
```

Comment:
this should go in integration-testing/install_dependencies.sh and here you should call that script. This way people can run integration-tests outside CI

Reply:
install_dependencies.sh is no more

Reply:
then what do I need to do in order to run the tests locally?